components/usage/pkg/apiv1/billing.go

```diff
@@ -23,50 +22,20 @@ import (
 	"gorm.io/gorm"
 )
 
-func NewBillingService(stripeClient *stripe.Client, billInstancesAfter time.Time, conn *gorm.DB, contentService contentservice.Interface) *BillingService {
+func NewBillingService(stripeClient *stripe.Client, conn *gorm.DB) *BillingService {
```

Comment:
Do we not want to keep the `billInstancesAfter` behaviour in the new reconciler?

Reply:
From the [discussion here](https://github.com/gitpod-io/gitpod/pull/12766#discussion_r965800503) it looks like not.